k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_config.tpl

```diff
@@ -62,16 +63,16 @@ cortx:
   rgw:
     iam:                                                            # DEPRECATED - IAM KEY
       endpoints:
-      - https://{{ .Values.cortxIoServiceName }}:8443
-      - http://{{ .Values.cortxIoServiceName }}:8000
+      - https://{{ $ioSvcName }}:8443
+      - http://{{ $ioSvcName }}:8000
     data:
       endpoints:
-      - http://{{ .Values.cortxIoServiceName }}:8000
-      - https://{{ .Values.cortxIoServiceName }}:8443
+      - http://{{ $ioSvcName }}:8000
+      - https://{{ $ioSvcName }}:8443
     s3:
       endpoints:
-      - http://{{ .Values.cortxIoServiceName }}:8000
-      - https://{{ .Values.cortxIoServiceName }}:8443
+      - http://{{ $ioSvcName }}:8000
```

Comment:
Need clarification on whether the service name is used for _any_ of these endpoints. In other words, what do these settings mean?